Client/Frontend/Strings.swift

```diff
@@ -532,9 +550,11 @@ extension Strings {
     public static let AppMenuShowTabsTitleString = MZLocalizedString("Menu.ShowTabs.Title", tableName: "Menu", value: "Show Tabs", comment: "Label for the button, displayed in the menu, used to open the tabs tray")
     public static let AppMenuSharePageTitleString = MZLocalizedString("Menu.SharePageAction.Title", tableName: "Menu", value: "Share Page With…", comment: "Label for the button, displayed in the menu, used to open the share dialog.")
     public static let AppMenuCopyURLTitleString = MZLocalizedString("Menu.CopyAddress.Title", tableName: "Menu", value: "Copy Address", comment: "Label for the button, displayed in the menu, used to copy the page url to the clipboard.")
+    public static let AppMenuCopyLinkTitleString = MZLocalizedString("Menu.CopyLink.Title", tableName: "Menu", value: "Copy Link", comment: "Label for the button, displayed in the menu, used to copy the current page link to the clipboard.")
```

Comment:
I thought we didn't wanna add tableName 😅

Reply:
@nbhasin2 😅 I was thinking about this. I think until we decide to get rid of the existing ones I thought I would continue keeping them grouped together to keep them consistent

Reply:
Ah true make sense I am good with this @boek 🙂